applications/MeshingApplication/python_scripts/mmg_process.py

```diff
@@ -706,6 +726,20 @@ def __generate_double_list_from_input(self,param):
 
       return double_list
 
+    def __generate_string_list_from_input(self,param):
```

Comment:
better use `GetStringArray`

Reply:
It is an auxiliar method, should I update the rest of methods? (for int and double)

Reply:
hm not sure
there is already `GetVector` for int and double

Up to you, but this method does exactly the same as GetStringArray

Reply:
True..., i will let like it is, and replace it in a mini PR